README.md

```diff
@@ -113,138 +86,195 @@ You can still find the latest installers in the [24.9.2 release](https://github.
 
-```bash
-echo ${HOME}/.condarc will be removed if it exists
-rm -f "${HOME}/.condarc"
-echo ${HOME}/.conda and underlying files will be removed if they exist.
-rm -fr ${HOME}/.conda
+```sh
+conda config --set auto_activate_base false
 ```
 
 ### Windows
 
-Download and execute the Windows installer. Follow the prompts, taking note of the options to
+Download and execute [the Windows installer](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe).
+Follow the prompts, taking note of the options to
 "Create start menu shortcuts" and "Add Miniforge3 to my PATH environment variable". The latter is
 not selected by default due to potential conflicts with other software. Without Miniforge3 on the
 path, the most convenient way to use the installed software (such as commands `conda` and `mamba`)
 will be via the "Miniforge Prompt" installed to the start menu.
 
+If desired, the
+`C:\Users\myusername\miniforge3\condabin` folder may be added to the path environment variable
+[manually](https://learn.microsoft.com/en-us/previous-versions/office/developer/sharepoint-2010/ee537574(v=office.14)#to-add-a-path-to-the-path-environment-variable)
+after installation so the software may be used more conveniently from any command prompt with limited chance of software conflicts.
+
 There are known issues with the usage of special characters and spaces in
 the installation location, see for example
 https://github.com/conda-forge/miniforge/issues/484.
 We recommend users install in a directory without any such characters in the name.
 
-### Non-interactive install
-
 For non-interactive usage one can use the batch install option:
 
-    bash Miniforge3-Linux-x86_64.sh -b  # or similar for other installers for unix platforms
+```sh
+start /wait "" Miniforge3-Windows-x86_64.exe /InstallationType=JustMe /RegisterPython=0 /S /D=%UserProfile%\Miniforge3
+```
 
-Look at the extra options by running the following:
+### Unix-like platforms (macOS & Linux)
 
-    bash Miniforge3-Linux-x86_64.sh -h
+Download the installer using curl or wget or your favorite program. For eg:
 
-or if you are on windows, run:
+```sh
+curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
+```
+
+or
+
+```sh
+wget "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
+```
 
-    start /wait "" Miniforge3-Windows-x86_64.exe /InstallationType=JustMe /RegisterPython=0 /S /D=%UserProfile%\Miniforge3
+Run the script with:
 
-### Downloading the installer as part of a CI pipeline
+```sh
+bash Miniforge3-$(uname)-$(uname -m).sh
+```
+
+or for non-interactive install (call with `-h` to list the extra options):
+
+```sh
+bash Miniforge3-$(uname)-$(uname -m).sh -b
+```
+
+### macOS with Homebrew
+
+On macOS, you can install miniforge with [Homebrew](https://brew.sh/) by running
+
+```sh
+brew install miniforge
+```
+
```

Comment:
I would rather remove this section. I'm happy to point to it, but I would rather omit the installation instructions and add a disclaimer that users should turn their issues to homebrew prior to asking us to do so.